c7n/resources/glue.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2016-2017 Capital One Services, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from botocore.exceptions import ClientError
+
+from c7n.manager import resources
+from c7n.filters import FilterRegistry
+from c7n.query import QueryResourceManager
+from c7n.utils import local_session, type_schema
+from c7n.actions import BaseAction
+from c7n.filters.vpc import SubnetFilter, SecurityGroupFilter
+
+
+@resources.register('glue-connection')
+class GlueConnection(QueryResourceManager):
+
+    class resource_type(object):
+        service = 'glue'
+        enum_spec = ('get_connections', 'ConnectionList', None)
+        detail_spec = None
+        id = 'Name'
+        name = 'Name'
+        date = 'CreationTime'
+        dimension = None
+        filter_name = None
+
+    filters = FilterRegistry('glue-connection.filters')
+    filter_registry = filters
```

Comment:
these two lines can be removed. the metaclass will put the filter_registry in place on class construction.